wit/bindgen/generator.go

```diff
@@ -47,8 +48,11 @@ type typeDecl struct {
 }
 
 type funcDecl struct {
-	f          function // The exported Go function
-	wasm       function // The wasmimport or wasmexport function
+	owner      wit.TypeOwner
+	dir        wit.Direction
+	f          *wit.Function
```

Comment:
suggest: add docs to describe what these fields do.

Reply:
Good idea. Will do